.babelrc

```diff
@@ -20,7 +20,6 @@
   },
   "plugins": [
     "babel-plugin-transform-class-properties",
-    "transform-object-rest-spread",
-    "external-helpers"
```

Comment:
- Issue: `ReferenceError: babelHelpers is not defined`
- Fix: This is because you're using '@babel/plugin-external-helpers', in your plugin list. If you do that, it is up to you to load the helpers. See [ref](https://github.com/babel/babel/issues/9366#issuecomment-455748788)